protoc-gen-go-bbs/go.mod

```diff
@@ -0,0 +1,5 @@
+module code.cloudfoundry.org/bbs/protoc-gen-go-bbs
+
+go 1.22.2
+
+require google.golang.org/protobuf v1.34.0
```

Comment:
Does this need a separate go.mod from diego-release's go.mod that applies to bbs due to different/conflicting versions of protobuf?

Reply:
Another old artifact that should be needed.

Reply:
Removed in [400d5a0](https://github.com/cloudfoundry/bbs/pull/109/commits/400d5a01bb8b412e2553e68d9df09ab99346350d)